### DISCUSSION TITLE
Using trusted header auth and defining admin email

### DISCUSSION THREAD

**Discussion by MadsLang** [open-webui/open-webui#6548] [Issues]
Hi!

I am deploying open-webui using a docker container on Google Cloud Run. I am setting authentication with trusted header using environment variables (WEBUI_AUTH_TRUSTED_EMAIL_HEADER). However, I cannot add an admin user. It seems like every user is authenticated with user role "user". 

I tried setting "ADMIN_EMAIL" or changing "DEFAULT_USER_ROLE" to "admin", but still users are added as "user". Is it possible to use trusted header auth, while defining the admin through environment variables?

**Comment by michaelwlt** [comment]
I found this in the docs. Might help you: https://docs.openwebui.com/tutorials/features/sso#oauth-role-management

**Comment by MadsLang** [comment]
I eventually found out that through Google Cloud Run, I mounted the volume in a storage bucket, and the webui.db was not getting overwritten by changing the environment variables. What I needed, was simply to erase the webui.db and building the container again. Losing the chat-history etc was not an issue for me here.
